consensus/src/epoch_manager.rs

```diff
@@ -398,7 +398,7 @@ impl EpochManager {
                 .context("[EpochManager] Verify event")
                 .map_err(|err|
                         // security log + return the error
-                        {send_struct_log!(security_log(security_events::CONSENSUS_INVALID_MESSAGE)
+                        {sl_error!(security_log(security_events::CONSENSUS_INVALID_MESSAGE)
                             .data("from_peer", &peer_id)
                             .data_display("error", &err)
                             .data("event", &unverified_event)
```

Comment:
Tbh, this formatting seems weird, but it was already there.  Was this the autoformatter?

Reply:
I made it nicer, that's one problem with rustfmt: it seems to allow different formatting to be valid : ( (unlike gofmt)

Reply:
Rust format doesn't seem to format nicely under some very specific nesting conditions.  I notice it doesn't like macros and we sometimes use macros for large blocks